app/services/import_polygons.rb

```diff
@@ -3,6 +3,18 @@ class ImportPolygons
   BUFFER_DISTANCES_IN_MILES = [5, 10, 15, 20, 25].freeze
   URL_MAXIMUM_LENGTH = 33_000
 
+  class ArcgisResponseError < StandardError
+    attr_reader :response_code
```

Comment:
Arguably this attr_reader should be a private method as you aren't trying to make it public. You can put `attr_reader` below a `private`.

Reply:
I've made some changes. Let me know what you think :)